<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingWindow</class>
 <widget class="QWidget" name="SettingWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>673</width>
    <height>458</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QScrollArea" name="scrollArea">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>651</width>
     <height>391</height>
    </rect>
   </property>
   <property name="widgetResizable">
    <bool>true</bool>
   </property>
   <widget class="QWidget" name="scrollAreaWidgetContents">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>649</width>
      <height>389</height>
     </rect>
    </property>
    <widget class="QCheckBox" name="boxAutoStart">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>131</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>设置开机自启动</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="boxSearchStartMenu">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>211</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>搜索开始目录文件夹</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QCheckBox" name="boxSearchRegistry">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>531</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>通过注册表查找安装的程序（不推荐，常用的软件已在菜单文件夹中存在）</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="boxSearchProgramFile">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>581</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>搜索Program File 文件夹来查找安装的程序（适合索引不在开始目录中的程序）</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnCustomDir">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>170</y>
       <width>181</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>设置自定义遍历目录</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnBannedDir">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>170</y>
       <width>181</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>设置目录黑名单</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="boxPreLoadResource">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>210</y>
       <width>311</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>设置启动预加载资源（程序启动会慢一些）</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="spResultItemNumber">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>260</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>10</number>
     </property>
     <property name="value">
      <number>4</number>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>260</y>
       <width>131</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string>设置搜索结果数量</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>390</x>
       <y>170</y>
       <width>161</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>遍历子目录的深度为1</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>390</x>
       <y>200</y>
       <width>231</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>只有目录的前缀完全匹配时，才不会遍历</string>
     </property>
     <property name="scaledContents">
      <bool>false</bool>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLineEdit" name="LEPlaceholderText">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>310</y>
       <width>421</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>440</x>
       <y>310</y>
       <width>171</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string>自定义搜索栏的提示文本</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="LEEmptyText">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>350</y>
       <width>421</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>440</x>
       <y>350</y>
       <width>181</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string>自定义搜索无结果的文本</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="boxSearchUWP">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>130</y>
       <width>541</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>搜索UWP应用</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="boxIgnoreUninstall">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>10</y>
       <width>121</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>忽略卸载程序</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QPushButton" name="btnConfirm">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>420</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>420</y>
     <width>501</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>软件更新地址：https://github.com/ghost-him/ZeroLaunch-CPP/releases</string>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
